administration/src/shared/verifications.ts

```diff
@@ -0,0 +1,21 @@
+import { ApplicationVerificationView } from '../generated/graphql'
+
+export const isEmailValid = (email: string): boolean => /^[^\s@]+@[^\s@]+\.[^\s@]+$/.test(email)
```

Comment:
🙃 this function is used not only for verifications but also to check applicant email (and can be potentially used to verify other emails, e.g. when creating a new admin), not sure if this function should be moved to "verifications.ts"

Reply:
To me this is a very generic utility function, I would place that in a file `/src/misc/emails.ts` or so.

What worries me more is the possibility that this regex will reject valid email addresses because it is hilariously simplified for what it claims to do. 

For the curious, a regex that correctly validates real world RFC822 email addresses looks like [this](https://pdw.ex-parrot.com/Mail-RFC822-Address.html), but we should use a specialized package, since it is really slow.